app/code/Magento/Catalog/Model/Category.php

```diff
@@ -866,7 +860,7 @@ public function getPathIds()
     public function getLevel()
     {
         if (!$this->hasLevel()) {
-            return count(explode('/', $this->getPath())) - 1;
+            return $this->getPath() ? (count(explode('/', $this->getPath())) - 1) : 0;
```

Comment:
Hello @karyna-tsymbal-atwix 
Please remove redundant brackets

```suggestion
            return $this->getPath() ? count(explode('/', $this->getPath())) - 1 : 0;
```

Reply:
done, I removed redundant brackets here and all places below that you mentioned